scripts/download_data.py

```diff
@@ -292,8 +300,24 @@ def downloadBams(self):
         escapeDir(3)
 
     def downloadFastas(self):
-        dirList = [self.dirName, 'references', self.referenceSetName]
+        dirList = [self.dirName, 'references']
         mkdirAndChdirList(dirList)
+        # Assemble reference set metadata
+        referenceSetMetadata = {
+            "assemblyId": 'TODO',
+            "description": 'TODO',
+            "isDerived": False,
+            "ncbiTaxonId": 9606,
+            "sourceAccessions": [],
+            "sourceUri": 'TODO',
+        }
+        referenceSetMetadataFilename = "{}.json".format(
+            self.referenceSetName)
+        dumpDictToFileAsJson(
+            referenceSetMetadata, referenceSetMetadataFilename)
+        # Download chromosomes
+        referenceSetDirList = [self.referenceSetName]
+        mkdirAndChdirList(referenceSetDirList)
```

Comment:
Bit confusing to make this variable just to wrap something in a list. How about:
```python
mkdirAndChdirList([self.referenceSetName])
```